OpenOversight/app/main/views.py

```diff
@@ -269,7 +267,6 @@ def officer_profile(officer_id):
             officer.image_url = url_for(
                 "static",
                 filename=faces[0].image.filepath.replace("/static/", ""),
-                _external=True,
```

Comment:
I'm guessing these are by default set to `True`?

Reply:
It should default to False in our use case here. But it is a good point, I probably should have set it to False to make it clear